softpack_core/schemas/environment.py

```diff
@@ -322,6 +322,8 @@ class Environment:
     state: Optional[State]
     tags: list[str]
     hidden: bool
+    cachedEnvs: list["Environment"] = None
+    envsUpdates: bool = True
```

Comment:
Not sure I can properly assess if this was set true and false everywhere it needs to be. Is there a better way to do this that avoids this concern?